gtfs-netex-test/aux_assertions.py

```diff
@@ -46,8 +46,9 @@ def process_assertions(assertions_file, input_file):
             else:
                 print(f'Assertion FAILed: XPath "{xpath_expression}" does not have more than {expected_count} results, was {len(results)}')
                 failed=1
-        else:
+        elif len(assertion.strip())>0:
```

Comment:
```suggestion
        elif len(assertion.strip()) > 0:
```
Maybe run something like ruff on the code, like happens with xsData.